tests/unit/compiler/test_source_map.py

```diff
@@ -101,6 +101,15 @@ def update_foo():
     assert "fallback function" in list(error_map.values())
 
 
+def test_error_map_with_user_error():
+    code = """
+@external
+def foo():
+    raise "some error"
+    """
+    error_map = compile_code(code, output_formats=["source_map"])["source_map"]["error_map"]
```

Comment:
this runs with optimizations on, right? the bug manifested when the optimizer optimized away the relevant instruction

Reply:
yeah, it optimizes assembly

Reply:
in the future it will be helpful to have some triage details in the PR description or commit message!

Reply:
> this runs with optimizations on, right? the bug manifested when the optimizer optimized away the relevant instruction

i looked into this but couldn't find that the optimizer is optimizing out the instruction. @cyberthirst do you still have your traige info available?